Pear/Settings/EditingViewController.swift

```diff
@@ -235,7 +230,12 @@ class EditingViewController: UIViewController {
                     return
                 }
                 let yoursAndMore = self.removeDuplicates(yourStrings: self.user.interests, moreStrings: result.data)
-                let stringToInterest = { ItemType.interest(Interest(name: $0, categories: nil, imageName: "")) }
+                let stringToInterest = {
```

Comment:
I prefer `interestOfString`, but maybe that's because I've been doing a lot of OCaml recently 🐪. It also mirrors your language from above (e.g. `setupSectionsFromInterests`)